Clarity over supported devices

woter1832 opened the pull request:
## Proposed change

Only a few devices are supported. Speakers are NOT supported. Change the wording to try and make it clearer.
<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.
-->



## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [X] Spelling, grammar or other readability improvements (`current` branch).
- [X] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

woter1832 committed:
Clarity over supported devices

Only a few devices are supported. Speakers are NOT supported.

frenck commented on source/_integrations/nest.markdown:
```diff
@@ -49,7 +49,7 @@ You are in control of the information and capabilities exposed to Home Assistant
 <div class='note'>
 
-The Nest Smart Device Management (SDM) API **requires a US$5 fee**.
+The Nest Smart Device Management (SDM) API **requires a US$5 fee**. BEFORE PAYING, make sure your device is [supported](https://developers.google.com/nest/device-access/supported-devices). Speakers are NOT included.
```
Full capitalized words are generally considered to be screaming. Let's not do that, as there is no reason for it either. Additionally, suggesting to make it similar to the change made above:

```suggestion
The Nest Smart Device Management (SDM) API **requires a US$5 fee**. Before buying, make sure your device is [supported](https://developers.google.com/nest/device-access/supported-devices).
```

woter1832 replied:
Hi @frenck, I've made an additional commit, using italics in lieu of capitals. (Although, if capitals = screaming, perhaps that's not a bad thing in this case).

frenck replied:
Screaming is always bad. There if not much to discuss in that regard.

woter1832 committed:
Merge branch 'home-assistant:current' into patch-1

woter1832 committed:
Further modification to SDM device clarification

Further to @frenck's comments, I have removed CAPITALs and used italics instead.

frenck commented on source/_integrations/nest.markdown:
```diff
@@ -49,7 +49,7 @@ You are in control of the information and capabilities exposed to Home Assistant
 <div class='note'>
 
-The Nest Smart Device Management (SDM) API **requires a US$5 fee**.
+The Nest Smart Device Management (SDM) API **requires a US$5 fee**. *Before buying*, make sure your device is [supported](https://developers.google.com/nest/device-access/supported-devices). Nest speakers (Mini and Audio) are *not* currently included.
```
Removal of unneeded emphasis. I also removed the additional line. The link already provides the list of devices supported.

```suggestion
The Nest Smart Device Management (SDM) API **requires a US$5 fee**. Before buying, make sure your device is [supported](https://developers.google.com/nest/device-access/supported-devices).
```

woter1832 committed:
Update source/_integrations/nest.markdown

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

frenck reviewed (approved):
Thanks, @woter1832 👍
